app/serializers/room_serializer.rb

```diff
@@ -1,10 +1,15 @@
 # frozen_string_literal: true
 
 class RoomSerializer < ApplicationSerializer
-  attributes :id, :name, :friendly_id, :created_at
+  attributes :id, :name, :friendly_id, :last_session
   attribute :shared_owner, if: -> { object.shared }
+  attribute :last_session, if: -> { object.last_session }
```

Comment:
Do we need this conditional rendering?

Reply:
Yes we need it for the case that the room has `last_session` as `nil` still.

I don't need to add it to line 4 though since I put line 6, therefore I will remove it from line 4 :+1:

Reply:
Why don't we still pass it through and then handle the nil case on the front-end?

Reply:
Well actually, you're already handling that case, so does this even make a difference?